src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -849,56 +1100,65 @@ public function providePathsForMakePathRelative()
             array('/usr/lib/symfony/', '/var/lib/symfony/src/Symfony/Component', '../../../../../../usr/lib/symfony/'),
             array('usr/lib/symfony/', 'var/lib/symfony/src/Symfony/Component', '../../../../../../usr/lib/symfony/'),
             array('/var/lib/symfony/src/Symfony/', '/var/lib/symfony/', 'src/Symfony/'),
-            array('/aa/bb', '/aa/bb', './'),
```

Comment:
Why are you changing these? Some changes you have done were already tested. E.g. you change here `/aa/bb` to `/aa/bb/`, but that one already exists. So much changes in tests is reason why everybody is hesitant to review this. Please go through your changes in tests and only change those that are faulty, otherwise just add new ones.